src/ceph-volume/ceph_volume/util/disk.py

```diff
@@ -872,6 +872,13 @@ def get_devices(_sys_block_path='/sys/block', device=''):
         else:
             metadata['device_nodes'] = devname
 
+        metadata['actuators'] = ""
```

Comment:
Python nitpick: here the default value is a string type.  However, below if the directory exists the actuators value becomes an int type. It's better to stick to one type, such as ints only. If a default of zero here is not appropriate I think `None` would be a better value. I think this would also be more type annotation friendly if they get added to this code sometime later.

Reply:
@phlogistonjohn Thanks for the feedback. This code can only prove presence of multiple actuators, not their absence, so it doesn't make sense to use zero as a default value here. An empty string was my choice, but let me think about your suggestion of using `None` instead. I'll have to test that.

You may also be interested in: https://github.com/ceph/ceph/pull/46344 .

Reply:
In mypy speak, using None means the type of the value would be `Optional[int]` and is more typical rather than `Union[str, int]` where the only str value is ever "".

Ah, sorry I didn't notice that the patch was not unique to this PR. Next time if I have feedback on this patch I'll make on the original PR.

Reply:
@phlogistonjohn I made the change. Do you have any other feedback for me to get this merged?